src/Function/Related/TypeIsomorphisms.agda

```diff
@@ -332,10 +331,10 @@ Related-cong {A = A} {B} {C} {D} A≈B C≈D =
 
 True↔ : ∀ {p} {P : Set p}
         (dec : Dec P) → ((p₁ p₂ : P) → p₁ ≡ p₂) → True dec ↔ P
-True↔ ( true because  [p]) irr =
-  inverse (λ _ → invert [p]) (λ _ → _) (λ _ → P.refl) (irr _)
-True↔ (false because ofⁿ ¬p) _ =
-  inverse (λ()) (invert (ofⁿ ¬p)) (λ ()) (⊥-elim ∘ ¬p)
+True↔ (yes p) irr =
+  inverse (λ _ → p) (λ _ → _) (λ _ → P.refl) (irr _)
+True↔ (no ¬p) _ =
+  inverse (λ()) ¬p (λ ()) (⊥-elim ∘ ¬p)
 
 ------------------------------------------------------------------------
```

Comment:
Same comment about strictness here as in commit f6ff90205b3b7176978de590b533e59226440fd2, but here it's more egregious, as the second pattern really is equivalent to `no ¬p` (only the `true`/`yes`/`lazy-yes` branch is potentially lazier...)

Reply:
And... resolved in favour of a lazy/lazier version in the same way, using a `let`-bound `invert`...